test/e2e/testsuites/dynamically_provisioned_pod_failover_tester.go

```diff
@@ -61,6 +68,26 @@ func (t *PodFailover) Run(client clientset.Interface, namespace *v1.Namespace, s
 		tDeployment.Exec(t.PodCheck.Cmd, t.PodCheck.ExpectedString)
 	}
 
+	//Check that AzVolumeAttachment resources were created correctly
+	//changes
+	time.Sleep(1 * time.Minute)
+	labelSelector := metav1.LabelSelector{MatchLabels: map[string]string{"disk.csi.azure.com/requested-role": "Replica"}}
+	azVolumeAttachments, err := t.AzDiskClient.DiskV1alpha1().AzVolumeAttachments(consts.AzureDiskCrdNamespace).List(context.Background(), metav1.ListOptions{LabelSelector: labels.Set(labelSelector.MatchLabels).String()})
+	if err != nil {
+		ginkgo.Fail("failed to get replica attachments")
+	}
+	maxShares, err := strconv.ParseInt(t.StorageClassParameters["maxShares"], 10, 0)
+	if len(azVolumeAttachments.Items) != int(maxShares-1) {
+		ginkgo.Fail("incorrect number of replica attachments found")
```

Comment:
It might be useful to include the expected and actual number of replicas in the message.